optuna/integration/mlflow.py

```diff
@@ -0,0 +1,97 @@
+import mlflow
+
+import optuna
+from optuna import type_checking
+
+if type_checking.TYPE_CHECKING:
+    from typing import Dict  # NOQA
+
+
+class MlflowCallback(object):
+    """Callback to track optuna trials with Mlflow.
+
+    This callback adds all relevant information that is
```

Comment:
How about removing `all` from this sentence? This is because there's missing information in current implementation such as `intermediate_values`. Even if those pieces of information will be added in the future, we still might want to exclude some information such as `system_attrs`.

Reply:
fixed